tensorflow/lite/java/demo/app/src/main/java/com/example/android/tflitecamerademo/Camera2BasicFragment.java

```diff
@@ -368,12 +368,7 @@ private void updateActiveModel() {
       classifier.setNumThreads(numThreads);
       if (device.equals(cpu)) {
       } else if (device.equals(gpu)) {
-        if (model.equals(mobilenetV1Quant)) {
```

Comment:
Do we also have to call setQuantizedModelsAllowed on the GpuDelegate?

Reply:
Yes, my bad. I thought it was enabled by default in related Java code. Updated.